test/E2ETests/E2ETests/Storage/BlobEndToEndTests.cs

```diff
@@ -99,7 +99,7 @@ public async Task BlobTrigger_Stream_Succeeds()
             {
                 logs = _fixture.TestLogs.CoreToolsLogs.Where(p => p.Contains(key));
                 return Task.FromResult(logs.Count() >= 1);
-            });
+            }, pollingInterval: 4000, timeout: 120 * 1000);
```

Comment:
Similar pattern used in [StorageHelpers.cs](https://github.com/Azure/azure-functions-dotnet-worker/blob/31f3a8b378dd2426764ece5a9dfa976d0085e6f5/test/E2ETests/E2ETests/Helpers/StorageHelpers.cs#L153)

This is added to deal with the following error -

```csharp
System.ApplicationException : Condition not reached within timeout.
```

Reply:
Let's also remove this, as we don't have evidence that the time here wasn't sufficient.

Reply:
As discussed, let's make sure we output the content of the test logger in case of failures as that will help us understand what that state was.